src/components/Player/Information/Text.js

```diff
@@ -2,10 +2,33 @@ import React, { Component } from 'react';
 
 class Text extends Component {
     render() {
+      const data = {
```

Comment:
I don't know if it helps, but you can use this if you like:

In a file named `data.json`:
```json
{
    "nick": "Silma",
    "points": 300,
    "rank": 4000,
    "onVacation": true,
    "underAttackProtection": false,
    "allianceID": 300
}
```

Then at the top of your file,

`import gameData from "./data.json";`

And that should give you a `gameData` variable equal to the contents of the `.json` file.

You should be able to find examples of this inside the AVARIAvs Rune I shared. I believe that's how we're pulling in the JSON files located in the `/src/store` folder.